data/mods/innawood/itemgroups/science_and_tech.json

```diff
@@ -0,0 +1,9 @@
+[
+  {
+    "type": "item_group",
+    "id": "science",
+    "ammo": 50,
+    "magazine": 100,
+    "items": [ { "group": "tools_science", "prob": 100 }, [ "pointy_stick", 50 ], [ "throwing_stick", 50 ], [ "spear_stone", 50 ] ]
```

Comment:
You sure you want to keep tools_science here?

Reply:
Um no. Thanks, difference between me planning to do a thing and actually doing it